app/router.ts

```diff
@@ -18,8 +19,29 @@ const Router = EmberRouter.extend({
         scheduleOnce('afterRender', this, () => {
             const page = this.get('url');
             const title = this.getWithDefault('currentRouteName', 'unknown');
-
-            this.get('metrics').trackPage({ page, title });
+            const metrics = this.get('metrics');
+            const {
+                authenticated,
+                isPublic,
+                resource,
+            } = config.metricsAdapters[0].dimensions;
+
+            /*
+              There's supposed to be a document describing how dimensions should be handled, but it doesn't exist yet.
+              When it does, we'll replace out this comment with the link to that documentation. For now:
+                  1) isPublic: Public, Private, or n/a (for pages that aren't covered by app permissions like the
+                  dashboard;
+                  2) authenticated: Logged in or Logged out
+                  3) resource: the JSONAPI type (node, file, user, etc) or n/a
+            */
+
+            metrics.trackPage({
+                [authenticated]: this.get('session.isAuthenticated') ? 'Logged in' : 'Logged out',
+                [isPublic]: title === 'file-detail' ? 'Public' : 'n/a',
```

Comment:
We can't rely on the title of the route to determine whether this is public or not. It has to come from the model in most cases (e.g. a project is public or private).

Reply:
Right, but the only file-detail we have right now is quickfiles, which are by definition public. When we have node file-details, we can have something more robust.

Reply:
I'd like to get this problem solved for the general case so that we have a pattern to follow going forward. The user-quickfiles route is also public. The dashboard is private. Those routes should be accounted for.

Reply:
According to the comment above the code, those routes are accounted for in the way that the ticket required (after consulting with Product and raising your concerns). I can make something more generic, but honestly anything particularly useful for our circumstances is going to need the GUID stuff the same way the resource does, because the things that have permissions are GUIDs.